src/Symfony/Bridge/PhpUnit/ClassExistsMock.php

```diff
@@ -77,7 +77,7 @@ public static function register($class)
         if (0 < strpos($class, '\\Tests\\')) {
             $ns = str_replace('\\Tests\\', '\\', $class);
             $mockedNs[] = substr($ns, 0, strrpos($ns, '\\'));
-        } elseif (0 === strpos($class, 'Tests\\')) {
+        } elseif (str_starts_with($class, 'Tests\\')) {
```

Comment:
`str_starts_with()` does not exist on PHP 7 which the PHPUit bridge still supports.

Reply:
Thanks for reviewing. I've reverted this change.